DependencyInjection/MonologExtension.php

```diff
@@ -755,11 +755,11 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                         }
                     }
 
-                    $hub = new Definition(
+                    $hubId = \sprintf('monolog.handler.%s.hub', $name);
+                    $hub = $container->setDefinition($hubId, new Definition(
                         'Sentry\\State\\Hub',
                         [new Reference($clientId)]
-                    );
-                    $container->setDefinition(\sprintf('monolog.handler.%s.hub', $name), $hub);
+                    ));
```

Comment:
This never worked correctly. The `monolog.handler.%s.hub` service was registered but never actually used. Instead, the entire definition was passed as an argument, resulting in the creation of two separate services: the unused `monolog.handler.%s.hub` service and an anonymous service that was passed to the handler.

Reply:
Can it be covered by a test?

Reply:
@GromNaN I've updated the [existing tests](https://github.com/symfony/monolog-bundle/compare/00cb0f9d61d6e96bdebb5b0c74c934a0e2a255f0..e2d05df7b3823cd904ecfc797bf1519c491e0c31).